lib/src/main/kotlin/com/swmansion/starknet/data/TypedData.kt

```diff
@@ -601,6 +605,14 @@ data class TypedData private constructor(
         fun fromJsonString(typedData: String): TypedData =
             Json.decodeFromString(serializer(), typedData)
     }
+
+    /**
+     * Create a JSON string from TypedData.
+     *
+     * @param typedData typed data instance
+     */
+    internal fun toJsonString(): String =
```

Comment:
Could you explain why this is an internal function?

Reply:
Actually no need to keep it internal, as we want to allow users to use it.